src/tasks/misc.ts

```diff
@@ -127,6 +128,14 @@ export const MiscQuest: Quest = {
       limit: { tries: 1 },
       freeaction: true,
     },
+    {
+      name: "Pantogramming",
+      ready: () => pantogramReady() && inCasual(),
```

Comment:
I would say just put the code directly as part of the task instead of calling to one-time methods in a different file.